core/src/Cuda/Kokkos_Cuda_Half.hpp

```diff
@@ -51,6 +51,11 @@
     !(defined(KOKKOS_ARCH_KEPLER) || defined(KOKKOS_ARCH_MAXWELL50) ||  \
       defined(KOKKOS_ARCH_MAXWELL52))
 #include <cuda_fp16.h>
+#include <iosfwd>  // istream & ostream for extraction and insertion ops
+#include <string>
+#include <Kokkos_NumericTraits.hpp>  // reduction_identity
+#include <Kokkos_Atomic.hpp>         // Kokkos::atomic_store
+#include <impl/Kokkos_Error.hpp>     // Kokkos::abort
```

Comment:
Are we using `Kokkos::abort` at all here?

Reply:
No. I will remove this include.